CHANGELOG.md

```diff
@@ -56,7 +63,7 @@ Now:
 Before:
 
 ```
-name: Household living in a 40 sq.meters accomodation while being free lodgers
+name: Household living in a 40 sq. metres accomodation while being free lodgers
```

Comment:
Seems to me `meter`is the US spelling and `metre` is the british-inspired spelling. I don't know what the rule is on the version of English to use :/

Reply:
Indeed, as I was told by @Br3nda, “`meter` is the US spelling, `metre` is everyone's else. A meter is the tool for measurement, a metre is the unit of distance.” :wink: That was actually the point of the initial commit from @verbman. The fact that native speakers feel compelled to open a PR for that is enough to justify the change for me.